apm-agent-plugins/apm-es-restclient-plugin/apm-es-restclient-plugin-common/src/main/java/co/elastic/apm/agent/esrestclient/RequestEndpointMap.java

```diff
@@ -0,0 +1,10 @@
+package co.elastic.apm.agent.esrestclient;
+
+import co.elastic.apm.agent.sdk.state.GlobalState;
+import co.elastic.apm.agent.sdk.weakconcurrent.WeakConcurrent;
+import co.elastic.apm.agent.sdk.weakconcurrent.WeakMap;
+
+@GlobalState
```

Comment:
I don't think this `@GlobalState` construct is needed anymore after you fix the plugin package naming.